src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background-color: $white-color;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  @include transition;
```

Comment:
The `@include transition;` mixin usage is ambiguous without seeing the mixin definition. Ensure that the mixin is correctly defined in `mixins.scss` and that it sets a reasonable default for the transition property, in accordance with the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).